src/st/no3/3C4EC.c

```diff
@@ -326,7 +326,230 @@ void EntityUnkId52(Entity* self) {
     }
 }
 
-INCLUDE_ASM("st/no3/nonmatchings/3C4EC", EntityCastleBridge);
+// func_psp_0923F328
+void EntityCastleBridge(Entity* self) {
+
+#define priorities D_801819A0
```

Comment:
Could we rename the actual variables? Add your new names to https://github.com/Xeeynamo/sotn-decomp/blob/master/config/symbols.us.stno3.txt